runtime/runtime/src/ext.rs

```diff
@@ -29,6 +29,20 @@ pub struct RuntimeExt<'a> {
     data_count: u64,
 }
 
+pub struct RuntimeExtValuePtr<'a> {
```

Comment:
Why don't we implement `ValuePtr` on `ValuePointer` directly, instead of wrapping it into a struct?

Also, we should probably pick names to better reflect the specialization, e.g. `ValuePointer` -> `TrieValuePointer` and `ValuePtr` -> `ValuePointer`.

Reply:
TrieValuePointer is defined in `near-store`, the trait is defined in `near-vm-logic` - wrapping is better than making one of them depend on another.

Renamed `ValuePointer` -> `TrieUpdateValuePointer`

Reply:
Then I suggest we wrap it as `pub struct RuntimeExtValuePtr<'a>(TrieUpdateValuePtr<'a>);` This is a common way to wrap stuff in Rust and it is known to be optimized out by Rust compiler.